frontend/benefit/handler/src/components/applicationReview/salaryBenefitCalculatorView/SalaryBenefitCalculatorView.tsx

```diff
@@ -372,10 +386,181 @@ const SalaryBenefitCalculatorView: React.FC<
           invalid={!!getErrorMessage(fields.endDate.name)}
           aria-invalid={!!getErrorMessage(fields.endDate.name)}
           errorText={getErrorMessage(fields.endDate.name)}
-          style={{ paddingRight: `${theme.spacing.xs}` }}
+          style={{ paddingRight: `${theme.spacing.s}` }}
         />
       </$GridCell>
 
+      {!isManualCalculator && (
+        <>
+          <$GridCell $colStart={1} $colSpan={11}>
+            <$CalculatorText
+              style={{ marginTop: theme.spacing.m, fontWeight: 'bold' }}
+            >
+              {t(`${translationsBase}.apprenticeshipCompensation`)}
+            </$CalculatorText>
+          </$GridCell>
+
+          {formik.values.trainingCompensations &&
+            formik.values.trainingCompensations.length > 0 && (
+              <>
+                <$GridCell $colStart={1} $colSpan={2}>
+                  <$CalculatorText>
+                    {fields.monthlyAmount.label}
+                  </$CalculatorText>
+                </$GridCell>
+
+                <$GridCell $colStart={3} $colSpan={3}>
+                  <$CalculatorText>
+                    {t(`${translationsBase}.apprenticeshipPeriod`)}
+                  </$CalculatorText>
+                </$GridCell>
+              </>
+            )}
+
+          {formik.values.trainingCompensations?.map((item) => (
```

Comment:
using `key` somewhere inside the map is adviced to make better performance: https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
This is now fixed